build/gulp/tasks/test-projects/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "import/no-unresolved": "off"
```

Comment:
We have requires for Enzyme, Simulant and other stuff that it's not in direct dependencies. Currently disabled